style.css

```diff
@@ -0,0 +1,81 @@
+@import "reset.css";
+@import "https://fonts.googleapis.com/css?family=Noto+Sans";
+
+* {
+  box-sizing: border-box;
+  margin: 16px, 15px;
+  padding: 5px;
+  color: black; 
+  font-size: 18px;
+  font-family: "Noto Sans", sans-serif;
+}
+
+body {
+  align-content: center;
+  width: 1024px;
+  margin: 0 auto;
+  background-color: #6a8585;		
+}
+
+section,
+header,
+footer {
+  width: 600px; 
+  margin: auto; 
+  border: 2px solid black; 
+  padding: 10px;
+  background-color: #c1d1d4;		
+}
+
+#photo,
+#top {
+  display: inline-block;
+}
+
+h1 {
+  text-align: center;
+  font-size: 2.5em;
+  padding: 5px;
+}
+
+h2 {
+  text-align: center;
+  font-size: 2rem;
+}
+
+h3 {
+  font-size: 1.5rem;
+  color:red;
+}
+
+li {
+  list-style-type: circle;
+  margin-left:20px;
+}
+
+footer>p {
+  font-size: 18px;
+}
+
+#top>p {
+  text-indent: 0.5em;
+  margin-top: 7px;
+}
+
+span {
+  font-weight: bold; 
+  color: red;
+  margin-left:0.5em;
+}
+
+p {
+  text-indent: 1.5em;
+}
+
+#top {
```

Comment:
It is better to use the class selector for styling, leaving the id for manipulations with JavaScript with DOM.